main.go

```diff
@@ -193,6 +198,15 @@ func findFiles(patterns []string) (_ []string, err error) {
 	return sortedFiles, err
 }
 
+func modify(filename string, content []byte) error {
+	return ioutil.WriteFile(filename, content, 0644)
+}
+
+func preview(filename string, originalContent []byte, modifiedContent []byte, comment []string, stderr io.Writer) error {
+	fmt.Fprintf(stderr, strings.Join(comment, "\n")+"\n")
```

Comment:
minor nit: we're joining the comments into a string (allocates a string) only to dump it to stdout. Could we use a for loop here instead? `fmt.Fprintln` in the for loop will obviate the need for the trailing "\n".